src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -214,7 +217,7 @@
   </data>
   <data name="MoveFocusWithArgCommandKey" xml:space="preserve">
     <value>Move focus {0}</value>
-    <comment>{0} will be replaced with one of the four directions "DirectionLeft", "DirectionRight", "DirectionUp", or "DirectionDown"</comment>
+    <comment>{0} will be replaced with one of the five directions "DirectionLeft", "DirectionRight", "DirectionUp", "DirectionDown" or "DirectionPrev"</comment>
   </data>

```

Comment:
the construction of this sentence will be wrong in most languages.

There is no general sentence structure that can support "Move focus left" and "Move focus previous" at the same time. Especially not in another language, right?

This is different from how we handled other commands, where we said (something like) `Move focus: direction=previous`. That looks like a straight parameter substitution.

This is different . . . it needs its own complete command string, "Move focus to last-used pane" or something.